src/CurrentUser.php

```diff
@@ -39,11 +44,13 @@ final class CurrentUser
     public function __construct(
         IdentityRepositoryInterface $identityRepository,
         EventDispatcherInterface $eventDispatcher,
-        ?SessionInterface $session = null
+        SessionInterface $session = null,
```

Comment:
Why replace `?SessionInterface` to `SessionInterface` ?

Reply:
And why there `?`, `null` can be passed there anyway.